apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayout.java

```diff
@@ -128,6 +128,11 @@ public static RifLayout parse(Workbook workbook, String sheetName) {
       if ("HHA".contains(sheetName) && "CLM_HHA_TOT_VISIT_CNT".equals(rifColumnName))
         rifColumnLength = Optional.of(4);
 
+      // Temporary Code for CCW Additions. These will go away
+      if ("HHA,Hospice".contains(sheetName) && "CLAIM_QUERY_CODE".equals(rifColumnName)) continue;
+      // Temporary Code for CCW Additions. These will go away
+      if ("Carrier".contains(sheetName) && "CARR_CLM_BLG_NPI_NUM".equals(rifColumnName)) continue;
```

Comment:
Are these going away before the merge to master? The when should be described here so we don't miss it. (If its further than "when we merge this to master eventually" it should also probably have a ticket and be linked here)